crates/optimism/txpool/src/supervisor/errors.rs

```diff
@@ -7,17 +94,237 @@ pub enum InteropTxValidatorError {
     #[error("inbox entry validation timed out, timeout: {0} secs")]
     Timeout(u64),
 
+    /// Message does not satisfy validation requirements
+    #[error(transparent)]
+    InvalidEntry(#[from] InvalidInboxEntry),
+
     /// Catch-all variant.
     #[error("supervisor server error: {0}")]
     Other(Box<dyn error::Error + Send + Sync>),
 }
 
+/// Invalid inbox entry
+#[derive(thiserror::Error, Debug)]
+pub enum InvalidInboxEntry {
+    /// Message does not meet minimum safety level
+    #[error("message does not meet min safety level, got: {got}, expected: {expected}")]
+    MinimumSafety {
+        /// Actual level of the message
+        got: SafetyLevel,
+        /// Minimum acceptable level that was passed to supervisor
+        expected: SafetyLevel,
+    },
```

Comment:
let's skip this variant, it belonged to the old interop spec which used method `check_ExecutingMessages` instead of the current `check_AccessList`